src/lib/layout/index.spec.ts

```diff
@@ -0,0 +1,261 @@
+import * as FS from 'fs-jetpack'
+import * as Path from 'path'
+import * as Layout from '.'
+import { getTmpDir } from '../../utils'
+import { rootLogger } from '../../utils/logger'
+import { MaybePromise } from '../utils'
+
+/**
+ * Disable logger timeDiff and color to allow snapshot matching
+ */
+rootLogger.settings({
+  pretty: {
+    timeDiff: false,
+    color: false,
+  },
+})
```

Comment:
To enable snapshot matching I need to disabled the `timeDiff`, otherwise the time diff differs at every run. However, while it works locally, it looks like in the CI, the `LOG_LEVEL` is set to `trace` and I can't find where this is done. Any idea?

Reply:
Let's jump on a call about this Monday, show me what's going on.